### PR TITLE
Remove duplication of runtime error logs

### PR DESCRIPTION
**Pull request by riyafa**
## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

**Review by warunalakshitha — APPROVED** [review]
LGTM

File: bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/util/RuntimeUtils.java [hunk 1]
```diff
@@ -79,7 +73,7 @@ public static void addToArray(Type type, String value, ArrayValue array) {
         // TODO: need to add parsing logic for ref values for both var args and other args as well.
         switch (type.getTag()) {
             case TypeTags.STRING_TAG:
-                array.add(array.size(), value);
+                array.add(array.size(), StringUtils.fromString(value));
```
Comment (warunalakshitha) [inline bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/util/RuntimeUtils.java:76]: Shall we write some tests to cover those missing lines?

**Review by warunalakshitha — CHANGES_REQUESTED** [review]

File: bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/util/RuntimeUtils.java (hunk 1 above)
Comment (riyafa) [reply bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/util/RuntimeUtils.java:76]: This is a refactoring I did. Not related to the main code

**Review by warunalakshitha — APPROVED** [review]
LGTM

**Pull request merged**